sdcm/mgmt/cli.py

```diff
@@ -23,7 +23,8 @@
 from distutils.version import LooseVersion
 
 import requests
-from invoke.exceptions import UnexpectedExit, Failure
+from invoke.exceptions import UnexpectedExit as InvokeUnexpectedExit, Failure as InvokeFailure
+from sdcm.remote.libssh2_client.exceptions import Failure as RemoteFailure
```

Comment:
to be aligned to other parts of the code that used it like this:
```
from sdcm.remote.libssh2_client.exceptions import UnexpectedExit as Libssh2UnexpectedExit
```

```suggestion
from sdcm.remote.libssh2_client.exceptions import Failure as Libssh2Failure
```

Reply:
Applied the change